TeachingRecordSystem/src/TeachingRecordSystem.Core/Dqt/QueryHandlers/GetQualificationByIdHandler.cs

```diff
@@ -0,0 +1,40 @@
+using Microsoft.PowerPlatform.Dataverse.Client;
+using Microsoft.Xrm.Sdk.Messages;
+using Microsoft.Xrm.Sdk.Query;
+using TeachingRecordSystem.Core.Dqt.Queries;
+
+namespace TeachingRecordSystem.Core.Dqt.QueryHandlers;
+
+public class GetQualificationByIdHandler : ICrmQueryHandler<GetQualificationByIdQuery, dfeta_qualification?>
+{
+    public async Task<dfeta_qualification?> Execute(GetQualificationByIdQuery query, IOrganizationServiceAsync organizationService)
+    {
+        var filter = new FilterExpression();
+        filter.AddCondition(dfeta_qualification.PrimaryIdAttribute, ConditionOperator.Equal, query.QualificationId);
```

Comment:
We probably want an `Active` filter on this too

Reply:
Might we have a future need to be able to toggle the status for a qualification in the TRS Console UI i.e. be able to show inactive quals too potentially?

Reply:
Done